src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+
```

Comment:
remove the empty line in all classes where it is present
```suggestion
```

Reply:
when i remove blank lines it fails maven test